test/acceptance/csi.bats

```diff
@@ -17,7 +17,7 @@ check_skip_csi() {
   kubectl create namespace acceptance
 
   # Install Secrets Store CSI driver
-  CSI_DRIVER_VERSION=1.0.0
+  CSI_DRIVER_VERSION=1.1.0
```

Comment:
I think this is the [driver version](https://github.com/kubernetes-sigs/secrets-store-csi-driver) (as opposed to our provider). Though it looks like it's up to 1.1.2 now.

```suggestion
  CSI_DRIVER_VERSION=1.1.2
```

Reply:
Ah, well that's confusing :) Thanks!